.github/workflows/pr-reminder.yaml

```diff
@@ -1,6 +1,7 @@
+---
 name: PR Review Reminder
 
-on:
+"on":
```

Comment:
Why is this quoted?

Reply:
on is considered a `bool` in yaml and so if it's not quoted - yaml linters yell